From patchwork Fri Jul 31 14:30:51 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Marcin Sloniewski <marcin.sloniewski@gmail.com>
X-Patchwork-Id: 11695067
Return-Path: 
 <SRS0=vkGX=BK=lists.infradead.org=linux-arm-kernel-bounces+patchwork-linux-arm=patchwork.kernel.org@kernel.org>
Received: from mail.kernel.org (pdx-korg-mail-1.web.codeaurora.org
 [172.30.200.123])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id B9F21722
	for <patchwork-linux-arm@patchwork.kernel.org>;
 Fri, 31 Jul 2020 14:31:27 +0000 (UTC)
Received: from merlin.infradead.org (merlin.infradead.org [205.233.59.134])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by mail.kernel.org (Postfix) with ESMTPS id 92601208E4
	for <patchwork-linux-arm@patchwork.kernel.org>;
 Fri, 31 Jul 2020 14:31:27 +0000 (UTC)
Authentication-Results: mail.kernel.org;
	dkim=pass (2048-bit key) header.d=lists.infradead.org
 header.i=@lists.infradead.org header.b="Lt3YNq+v";
	dkim=fail reason="signature verification failed" (2048-bit key)
 header.d=gmail.com header.i=@gmail.com header.b="rooF5Y/L"
DMARC-Filter: OpenDMARC Filter v1.3.2 mail.kernel.org 92601208E4
Authentication-Results: mail.kernel.org;
 dmarc=fail (p=none dis=none) header.from=gmail.com
Authentication-Results: mail.kernel.org;
 spf=none
 smtp.mailfrom=linux-arm-kernel-bounces+patchwork-linux-arm=patchwork.kernel.org@lists.infradead.org
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=merlin.20170209; h=Sender:Content-Transfer-Encoding:
	Content-Type:Cc:List-Subscribe:List-Help:List-Post:List-Archive:
	List-Unsubscribe:List-Id:MIME-Version:Message-Id:Date:Subject:To:From:
	Reply-To:Content-ID:Content-Description:Resent-Date:Resent-From:Resent-Sender
	:Resent-To:Resent-Cc:Resent-Message-ID:In-Reply-To:References:List-Owner;
	bh=5bfzbNe1z8q22R2VERGYLDMIQrfP7PggkAr/XL6D3mw=; b=Lt3YNq+vn+BoHN23MO3PZGlHVm
	dqrgDQoV1elTWeHOcV6CkwIlovqZi8lwX+gBgjtKkkV+zYUS/kZiL+xI2L6R99cHzyqzJs/1PWlmJ
	27ySLX2cDxUVywgtQHJevjS9X6HqxUcwcU0K0pjpGimxqORMfVAU+Z4oRAlFTxgPs0sv2YaqhJmvk
	GMjjg/yUqXA9nc6/vRZLWs2eOj/gPJ8MHa9IW25qB6cSgaVNlSTScEdZcC1TOyRW3zRW9WXFS/aO7
	hOjOksgfh5/LdBQK58rQ8ET42v8hmowPUckNRoLYOeScRel0omz6J3Cn/lvDHdr+ghoXw0ofj0nKR
	qitF9cJQ==;
Received: from localhost ([::1] helo=merlin.infradead.org)
	by merlin.infradead.org with esmtp (Exim 4.92.3 #3 (Red Hat Linux))
	id 1k1W43-0007vm-3P; Fri, 31 Jul 2020 14:31:15 +0000
Received: from mail-ej1-x641.google.com ([2a00:1450:4864:20::641])
 by merlin.infradead.org with esmtps (Exim 4.92.3 #3 (Red Hat Linux))
 id 1k1W3z-0007uF-H5
 for linux-arm-kernel@lists.infradead.org; Fri, 31 Jul 2020 14:31:12 +0000
Received: by mail-ej1-x641.google.com with SMTP id f24so11136917ejx.6
 for <linux-arm-kernel@lists.infradead.org>;
 Fri, 31 Jul 2020 07:31:09 -0700 (PDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=gmail.com; s=20161025;
 h=from:to:cc:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=TnSVvBPLgxOr7Okxa4+yFdbMwIKkVndKbRuepRfY/Zs=;
 b=rooF5Y/LejVp6e+lkheBsBcOKwrWhHZzYXHf1m9XAOFNv87fHiPOh8t6nVxb4zVyuN
 QDpyFEFyiTwlk8xKArO+TX4KUIY++pi8GQ2aMGQeJFu49tn7aim8G/kueOGS8tfEIrpQ
 m8Eb/crBpL+aHjgty9b9jP36O8almuZhO0THmWfwhZOGRGmaQuoebyiVPkJPoZnWfb12
 IcMNREwMZeqv+FGO3XG63F7LBQcdiGEnQayFjSUl4e7jJzqbqQliGxVniSm03H8C3sTg
 +qszh99IXLCLTlEOsKO0vTmCHnKjuYFc0sYdgnG4a917/G1WL5EJUUsElxaTKZz5mVRc
 1i7g==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
 d=1e100.net; s=20161025;
 h=x-gm-message-state:from:to:cc:subject:date:message-id:mime-version
 :content-transfer-encoding;
 bh=TnSVvBPLgxOr7Okxa4+yFdbMwIKkVndKbRuepRfY/Zs=;
 b=A2u1MUjMrLVbjxjdbxnHDqDlmYaHkw8q9FotqDJF3dCvdJvvyU2NxIeq5/j6zq3jpO
 j31RaU/DPYHiSg5kplfdrgkSOZK+K0lTavU+S+ywyshX/2yCVI8nCl9OzuMkEov8bT/m
 43QDejJpL/SmgZRKu+LNQ8zDDpsj1NshF8wDd4xA0IKyL+kagoN5c00qIq3a6sIIrRJ2
 pgcyJ2akaa4WaIZIWOEMgTbYRLKi50FZSaPZWv90+dWNZxqeofPSMfNUkMWaB2FDVP/7
 34cNkN4DkZW4PAiVWgnCAFOhW+uzvh/bb9ur5uuVgPHZlhSAXK7k6MxSqtGByrv5anj4
 9AbQ==
X-Gm-Message-State: AOAM533qdoXA7OhRKnodwjuFemgbnuzB6M1Ezj5dK16V2RmuS2yAoxoY
 Fh9bBRKH+4RTO9BMpR49AGXz1Asvs5w=
X-Google-Smtp-Source: 
 ABdhPJybf/XZgZB/Q/5YD/PtnotUkX1ldsL6Uu0I4qModtm+zX/Ggq7KmPlZXxlfNH2FA8a1tV0ngA==
X-Received: by 2002:a17:906:269a:: with SMTP id
 t26mr4331783ejc.286.1596205867955;
 Fri, 31 Jul 2020 07:31:07 -0700 (PDT)
Received: from blackhead.home ([2a01:112f:a1c:7900:7316:ce1e:7b0b:6bd7])
 by smtp.gmail.com with ESMTPSA id t19sm9749213edw.63.2020.07.31.07.31.05
 (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
 Fri, 31 Jul 2020 07:31:06 -0700 (PDT)
From: Marcin Sloniewski <marcin.sloniewski@gmail.com>
To: linux-arm-kernel@lists.infradead.org
Subject: [PATCH v6 1/3] dt-bindings: vendor-prefixes: add Seeed Studio
Date: Fri, 31 Jul 2020 16:30:51 +0200
Message-Id: <20200731143053.44866-1-marcin.sloniewski@gmail.com>
X-Mailer: git-send-email 2.27.0
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20200731_103111_656264_F6F92A11 
X-CRM114-Status: GOOD (  11.66  )
X-Spam-Score: -0.2 (/)
X-Spam-Report: SpamAssassin version 3.4.4 on merlin.infradead.org summary:
 Content analysis details:   (-0.2 points)
 pts rule name              description
 ---- ----------------------
 --------------------------------------------------
 -0.0 RCVD_IN_DNSWL_NONE     RBL: Sender listed at https://www.dnswl.org/,
 no trust [2a00:1450:4864:20:0:0:0:641 listed in]
 [list.dnswl.org]
 0.0 FREEMAIL_FROM          Sender email is commonly abused enduser mail
 provider [marcin.sloniewski[at]gmail.com]
 -0.0 SPF_PASS               SPF: sender matches SPF record
 0.0 SPF_HELO_NONE          SPF: HELO does not publish an SPF Record
 -0.1 DKIM_VALID_EF          Message has a valid DKIM or DK signature from
 envelope-from domain
 0.1 DKIM_SIGNED            Message has a DKIM or DK signature,
 not necessarily
 valid
 -0.1 DKIM_VALID Message has at least one valid DKIM or DK signature
 -0.1 DKIM_VALID_AU          Message has a valid DKIM or DK signature from
 author's domain
X-BeenThere: linux-arm-kernel@lists.infradead.org
X-Mailman-Version: 2.1.29
Precedence: list
List-Id: <linux-arm-kernel.lists.infradead.org>
List-Unsubscribe: 
 <http://lists.infradead.org/mailman/options/linux-arm-kernel>,
 <mailto:linux-arm-kernel-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-arm-kernel/>
List-Post: <mailto:linux-arm-kernel@lists.infradead.org>
List-Help: <mailto:linux-arm-kernel-request@lists.infradead.org?subject=help>
List-Subscribe: 
 <http://lists.infradead.org/mailman/listinfo/linux-arm-kernel>,
 <mailto:linux-arm-kernel-request@lists.infradead.org?subject=subscribe>
Cc: robh@kernel.org, a.fatoum@pengutronix.de, alexandre.torgue@st.com,
 stephan@gerhold.net, mani@kernel.org, heiko.stuebner@theobroma-systems.com,
 linus.walleij@linaro.org, linux-kernel@vger.kernel.org, lkundrak@v3.sk,
 marcin.sloniewski@gmail.com, robh+dt@kernel.org, broonie@kernel.org,
 mcoquelin.stm32@gmail.com, allen.chen@ite.com.tw, sam@ravnborg.org,
 linux-stm32@st-md-mailman.stormreply.com, devicetree@vger.kernel.org
Sender: "linux-arm-kernel" <linux-arm-kernel-bounces@lists.infradead.org>
Errors-To: 
 linux-arm-kernel-bounces+patchwork-linux-arm=patchwork.kernel.org@lists.infradead.org

Add the "seeed" vendor prefix for Seeed Technology Co., Ltd
Website: https://www.seeedstudio.com/

Signed-off-by: Marcin Sloniewski <marcin.sloniewski@gmail.com>
Acked-by: Rob Herring <robh@kernel.org>
---
 Documentation/devicetree/bindings/vendor-prefixes.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.yaml b/Documentation/devicetree/bindings/vendor-prefixes.yaml
index 9aeab66be85f..7dd03b3e9d3c 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.yaml
+++ b/Documentation/devicetree/bindings/vendor-prefixes.yaml
@@ -902,6 +902,8 @@ patternProperties:
     description: Schindler
   "^seagate,.*":
     description: Seagate Technology PLC
+  "^seeed,.*":
+    description: Seeed Technology Co., Ltd
   "^seirobotics,.*":
     description: Shenzhen SEI Robotics Co., Ltd
   "^semtech,.*":
